{{ cookiecutter.__package_name_kebab_case }}/Dockerfile

```
@@ -32,13 +24,33 @@ RUN groupadd --gid $GID user && \
 USER user
 
 # Create and activate a virtual environment.
-RUN python -m venv /opt/{{ cookiecutter.__package_name_kebab_case }}-env
-ENV PATH /opt/{{ cookiecutter.__package_name_kebab_case }}-env/bin:$PATH
 ENV VIRTUAL_ENV /opt/{{ cookiecutter.__package_name_kebab_case }}-env
+ENV PATH $VIRTUAL_ENV/bin:$PATH
+RUN python -m venv $VIRTUAL_ENV
 
 # Set the working directory.
 WORKDIR /workspaces/{{ cookiecutter.__package_name_kebab_case }}/
 
+FROM base as poetry
```

I just remembered we separated each stage by 3 blank lines (once suggested by Ruben to help see when a new stage begins):
```suggestion


FROM base as poetry
```

---

Fixed